doc/user/layouts/shortcodes/integrations/mz-debug/memory-profiles.md

```diff
@@ -0,0 +1,5 @@
+### Memory profiles
+
+`mz-debug` outputs heap profiles if
```

Comment:
Depending on what we want to emphasize, we could flip these sentences as (something like)

By default, `mz-debug` outputs heap profiles for each service in `profiles/{service}.memprof.pprof.gz`. To turn off this behavior, you can set [`--dump-heap-profiles`](#dump-heap-profiles) to false.

Reply:
Also, where  are these configs doc'ed?  I don't see them on the page.

Reply:
I do see them in the main branch version's mz_debug_option.yml though... just an oversight on the self-managed-docs branch

Reply:
I wonder... maybe it's worth it to just remove these from the self managed docs and to centralize in the main cloud docs?